zulipterminal/ui_tools/views.py

```diff
@@ -1410,6 +1415,8 @@ def keypress(self, size: urwid_Size, key: str) -> str:
                 message_links=self.message_links,
                 time_mentions=self.time_mentions,
             )
+        elif is_command_key("VIEW_IN_BROWSER", key):
+            self.controller.view_in_browser(self.msg["id"])
```

Comment:
I'd suggest adding the functionality first, then hooking it up in a separate commit, which here could combine the small first commit with the UI which uses it.
(new popups tend to have a more complex structure since a new view often requires the key first)